capstone-frontend/src/components/ClubPage.jsx

```diff
@@ -1,10 +1,27 @@
 import React, { Component } from 'react';
 
 class ClubPage extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      response: {}
+    }
+  }
+
+  fetchClubData = () => {
+    fetch(`/api/clubs?id=${this.props.match.params.id}`)
+        .then(response => response.json()).then(res => this.setState({response: res[0]}));
+  }
+
   render() {
+    this.fetchClubData();
     return (
       <div>
-        <div> {this.props.match.params.id} </div>
+        <div> Club Name: {this.state.response.name} </div>
+        <div> ID: {this.props.match.params.id} </div>
+        <div> Description: {this.state.response.description} </div>
+        <div> OwnerID: {this.state.response.ownerID} </div>
+        <div> GbookID: {this.state.response.gbookID} </div>
```

Comment:
Not necessary to do right now since this is basically just dumping the data from the API onto the DOM, but don't forget to add tests for this component once you flesh it out!

Reply:
Will do! I didn't bother adding testing here since I knew this was just a temporary display and would change a ton, but will definitely incorporate in the future.